GUI/src/components/HistoricalChat/ChatMessage.tsx

```diff
@@ -1,23 +1,31 @@
-import { FC } from 'react';
+import React, { FC } from 'react';
 import { format } from 'date-fns';
 
 import { Message } from 'types/message';
+import Linkifier from "../Chat/linkifier";
 
 type ChatMessageProps = {
   message: Message;
   onMessageClick?: (message: Message) => void;
 }
 
 const ChatMessage: FC<ChatMessageProps> = ({ message, onMessageClick }) => {
-  return (
-    <div className='historical-chat__message'>
-      <div className='historical-chat__message-text'
-           onClick={onMessageClick ? () => onMessageClick(message) : undefined}>{message.content}</div>
-      <time dateTime={message.authorTimestamp} className='historical-chat__message-date'>
-        {format(new Date(message.authorTimestamp), 'HH:ii:ss')}
-      </time>
-    </div>
-  );
+    return (
+        <div className="historical-chat__message">
+            <div
+                className="historical-chat__message-text"
+                onClick={onMessageClick ? () => onMessageClick(message) : undefined}
```

Comment:
**suggestion:** imo it's better to handle empty click agains `null` not `undefined`

Reply:
Then ill get type complains